packages/vats/src/lib-board.js

```diff
@@ -28,28 +61,112 @@ const calcCrc = (data, crcDigits) => {
 };
 
 /**
- * Create a board to post things on.
+ * Make board state init function, closing over baggage.
  *
- * @param {bigint | number} [initSequence]
- * @param {object} [options]
- * @param {string} [options.prefix]
- * @param {number} [options.crcDigits]
+ * @see {prepareExoClassKit}
+ * @see {@link packages/SwingSet/docs/virtual-objects.md|SwingSet Virtual Objects}
+ *
+ * Hoisting this function makes defining the state type concise.
+ * @typedef {ReturnType<ReturnType<makeInit>>} BoardState
+ *
+ * @param {import('@agoric/vat-data').Baggage} baggage
+ *
+ *  typedef {import('@agoric/store').Key} Key  // XXX Why doesn't this work?
+ * @typedef {import('@endo/marshal').Passable} Key
+ */
+const makeInit =
+  baggage =>
+  /**
+   * @param {bigint | number} [initSequence]
+   * @param {object} [options]
+   * @param {string} [options.prefix]
+   * @param {number} [options.crcDigits]
+   */
+  (
+    initSequence = 0,
+    { prefix = DEFAULT_PREFIX, crcDigits = DEFAULT_CRC_DIGITS } = {},
+  ) => {
+    // XXX passing these options around in { state } seems somewhat costly,
```

Comment:
`state` contains data unique the exo, whether it's mutable or immutable. what would be less costly?

Reply:
What would be less costly is taking these completely unused (and untested) options out of the API.

Reply:
ah, I misread "we never vary them". SGTM to take out the options and remove the `DEFAULT_`. They used to be zero cost YUMNI (you, uh, might need it) but now they have real runtime costs.

Reply:
turns out there _is_ a test for the `prefix` option. might as well leave them all as-is.